src/blueapi/service/runner.py

```diff
@@ -98,15 +100,17 @@ def stop(self):
             self._state = EnvironmentResponse(
                 environment_id=environment_id,
                 initialized=False,
-                error_message=self._state.error_message,
+                error_message=_safe_message(
+                    type(self).__name__, self._state.error_message
+                ),
```

Comment:
Should: Since this just propagates the previous error message it will become `WorkerDispatcher: <previous message>`, where it was `<previous message>` before. That feels very inconsistent, the user should see the same message when the same thing breaks. Since this is always set by `_safe_exception_message` you're probably safe just propagating it without checking?